### DISCUSSION TITLE
Add RRule plugin support

### DISCUSSION THREAD

**Discussion by Larry-Home** [saade/filament-fullcalendar#89] [Ideas]
We need to be able to support complex rules and have decided to use RRule plugin support.  Our model has the fields and provides it to the calendar on our site, but we would like the Filament widget to display this as well.  The steps for including are shown at:

https://fullcalendar.io/docs/v5/rrule-plugin

Thanks for your consideration!

**Comment by saade** [comment]
Hello! I don't use this plugin, so i don't know how it should work or behave,

Feel free to submit a PR with this contribution!

**Comment by Larry-Home** [comment]
I have submitted the PR to add this plugin - https://github.com/saade/filament-fullcalendar/pull/90.  Thanks!

**Comment by saade** [comment]
thank you
